testament/categories.nim

```diff
@@ -262,6 +262,8 @@ proc debuggerTests(r: var TResults, cat: Category, options: string) =
   if fileExists("tools/nimgrep.nim"):
     var t = makeTest("tools/nimgrep", options & " --debugger:on", cat)
     t.spec.action = actionCompile
+    # force target to C because of MacOS 10.15 clang++ bug (see #15612)
```

Comment:
make it more precise using comment link https://github.com/nim-lang/Nim/pull/15612#issuecomment-712471879 ; or even better, file a new bug and refer to it

Reply:
Site https://bugreport.apple.com/ does not open for me.
I left feedback in https://feedbackassistant.apple.com/feedback/8825785, but I'm not sure whether it's publicly available or not?
So I will insert the comment link.